CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Added
+
+#### From [#605](https://github.com/polkadot-fellows/runtimes/pull/605)
+
+- Now each system extension has its own weight, defined by `ExtensionWeightInfo`.
+- Parachains define the default `CoreSelector` strategy. 
```

Comment:
Let's add a bit of detail for the core selector, it was introduced with the [RFC103 implementation ](https://github.com/polkadot-fellows/RFCs/blob/main/text/0103-introduce-core-index-commitment.md).